frontend/src/components/common/Filter/components/FilterContent/components/FilterViews/components/FilterView/index.tsx

```diff
@@ -7,15 +8,14 @@ import {
 } from "src/components/common/Filter/common/entities";
 import {
   ViewDivider,
-  ViewHeader,
   ViewPanel,
   ViewPanelScroll,
 } from "src/components/common/Filter/components/FilterContent/components/FilterViews/components/FilterView/style";
 import FilterViewList from "src/components/common/Filter/components/FilterContent/components/FilterViews/components/FilterViewList";
 import FilterSearch from "src/components/common/Filter/components/FilterSearch";
 import { useFilterSearch } from "src/components/common/Filter/components/FilterSearch/common/useFilterSearch";
 
-const ADDITIONAL_PANEL_WIDTH = 8;
+const ADDITIONAL_PANEL_WIDTH = 24;
```

Comment:
this constant seems to have the same value as `ADDITIONAL_MENU_WIDTH ` in `frontend/src/components/common/Filter/components/FilterContent/components/FilterMenu/index.tsx`. Is it a coincidence or something that can shared between these two components?

Reply:
Nice pick up @tihuan - I've removed `ADDITIONAL_PANEL_WIDTH` and using `ADDITIONAL_MENU_WIDTH` for both instances.